### DISCUSSION TITLE
Gosund WP2 - New Device Database entry

### DISCUSSION THREAD

**Discussion by awkimball** [RavenSystem/esp-homekit-devices#2286] [MEPLHAA Scripts Collection]
Hi,

I have recently installed HAA on some Gosund WP2 dual outlets and wanted to share the script that I came up with. I tried to fork and make a PR to the wiki but it doesn't appear that I'm able to do that.

Script: 
`{"c":{"io":[[[12,14],2],[[2,13,1],2],[[0,4],6,1]],"b":[[0,5]],"l":1,"s":0},"a":[{"t":2,"0":{"r":[[14,0],[13,1]]},"1":{"r":[[14,1],[13,0]]},"b":[[0,1]],"es":[{"t":2,"0":{"r":[[12,0],[2,1]]},"1":{"r":[[12,1],[2,0]]},"b":[[4,1]]}]}]}`

GPIO details:
- GPIO14: Relay1  
- GPIO12: Relay2  
- GPIO13: Led1  
- GPIO2: Led2  
- GPIO0: Button1  
- GPIO4: Button2  
- GPIO1: StatusLED  
- Hold Top button for 8 sec to enter Setup Mode

Database entry line: 
``|Gosund|WP2|ESP8266|Outlet|GPIO14: Relay1  GPIO12: Relay2  GPIO13: Led1  GPIO2: Led2  GPIO0: Button1  GPIO4: Button2  GPIO1:StatusLED  Hold Top button for 8 sec to enter Setup Mode|`{"c":{"io":[[[12,14],2],[[2,13,1],2],[[0,4],6,1]],"b":[[0,5]],"l":1,"s":0},"a":[{"t":2,"0":{"r":[[14,0],[13,1]]},"1":{"r":[[14,1],[13,0]]},"b":[[0,1]],"es":[{"t":2,"0":{"r":[[12,0],[2,1]]},"1":{"r":[[12,1],[2,0]]},"b":[[4,1]]}]}]}`|``

**Comment by xrust83** [comment]
https://github.com/RavenSystem/esp-homekit-devices/discussions/1506

**Comment by RavenSystem** [comment]
Many thanks!

I added script to the Wiki. PRs don't work with Wiki pages.
